cmake/CMakeLists.txt

```diff
@@ -367,6 +368,11 @@ if (onnxruntime_USE_ROCM)
   if (onnxruntime_USE_COMPOSABLE_KERNEL AND ROCM_VERSION_DEV VERSION_LESS "5.3")
     message(WARNING "composable kernel is only supported on ROCm >= 5.3")
     set(onnxruntime_USE_COMPOSABLE_KERNEL OFF)
+    set(onnxruntime_USE_COMPOSABLE_KERNEL_CK_TILE OFF)
+  endif()
+  if (onnxruntime_USE_COMPOSABLE_KERNEL_CK_TILE AND ROCM_VERSION_DEV VERSION_LESS "5.3")
+    message(FATAL_ERROR "ck_tile can only be enabled on ROCm >= 5.0 due to compilation speed")  # FIXME:
```

Comment:
ROCm >= 5.3

Reply:
Fixed. This should be altered to `6.0` actually, the ck library does not actively maintain backward compiler compatibability.